WarehouseConfiguration warehouse_configuration
---
bool success
